Formula/m/media-control.rb

```diff
@@ -0,0 +1,43 @@
+class MediaControl < Formula
+  desc "Control and observe media playback from the command-line"
+  homepage "https://github.com/ungive/media-control"
+  # pull from git tag to get submodules
+  url "https://github.com/ungive/media-control.git",
+      tag:      "v0.2.0",
+      revision: "f6c7293a8c6199f9ccc07c7d650737fb5d788001"
+  license "BSD-3-Clause"
+  head "https://github.com/ungive/media-control.git", branch: "master"
+
+  no_autobump! because: :requires_manual_review
+
```

Comment:
it should be removed?

Reply:
Done! I've added it just in case because many other formulas seem to have it.